src/commands/createSwaConfigFile.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { writeFile } from 'fs-extra';
+import { join } from 'path';
+import { EntryInfo, promise } from 'readdirp';
+import { TextDocument, Uri, window, workspace } from 'vscode';
+import { ext } from 'vscode-azureappservice/out/src/extensionVariables';
+import { IActionContext } from 'vscode-azureextensionui';
+import { configFileName } from '../constants';
+import { localize } from '../utils/localize';
+import { getWorkspaceFolder } from '../utils/workspaceUtils';
+
+export async function createSwaConfigFile(context: IActionContext): Promise<void> {
+    const workspacePath: string = (await getWorkspaceFolder(context)).uri.fsPath;
```

Comment:
We also might be able to use their .yml config file (if they have it) to put the file in the app_location folder.

Reply:
Feels like you should let the user pick where the file goes, maybe a quick pick and/or browse dialog